gax-grpc/src/main/java/com/google/longrunning/OperationsClient.java

```diff
@@ -106,11 +103,9 @@
  * </pre>
  */
 @Generated("by GAPIC")
-@ExperimentalApi
+@BetaApi
```

Comment:
It's not in your change here so I can't comment on the correct line, but we should update the generated sample code above which shows how to customize the credentials.